src/invidious/routes/api/v1/search.cr

```diff
@@ -55,4 +55,30 @@ module Invidious::Routes::API::V1::Search
       return error_json(500, ex)
     end
   end
+
+  def self.hashtag(env)
+    hashtag = env.params.url["hashtag"]
+
+    # page does not change anything.
+    # page = env.params.query["page"]?.try &.to_i?|| 1
+
+    page = 1
+    locale = env.get("preferences").as(Preferences).locale
+    region = env.params.query["region"]?
+    env.response.content_type = "application/json"
+
+    begin
+      results = Invidious::Hashtag.fetch(hashtag, page, region)
+    rescue ex
+      return error_json(400, ex)
+    end
+
+    JSON.build do |json|
+      json.array do
```

Comment:
should I make an `items` property to contain the videos incase we decide to show more hashtag information in the future or decide to use continuation tokens?

Reply:
Yes, that's a good idea! Maybe `results` would be more fitting, though?